build/pipelines/templates/build-console-ci.yml

```diff
@@ -11,6 +11,7 @@ jobs:
     BuildConfiguration: ${{ parameters.configuration }}
     BuildPlatform: ${{ parameters.platform }}
     WindowsTerminalBranding: ${{ parameters.branding }}
+    EnableRichCodeNavigation: true

```

Comment:
I don't see anything using this variable.

Reply:
apparently it's magic to turn it on in the code nav tooling itself.

Reply:
Yes, we use this parameter to tell us to run a specific package called RichCodeNav.EnvVarDump, which is needed for some builds so we can get information about the environment the build is running in.